lib/kits/core-ui/components/common/offer-overview.ts

```diff
@@ -206,6 +197,36 @@ const OfferOverview = ({
         addShoppingListListener();
     };
 
+    const formatOfferDate = (dateStr) => {
+        const date = parseDateStr(dateStr);
+        const month = `0${date.getUTCMonth() + 1}`.slice(-2);
+        const day = `0${date.getUTCDate()}`.slice(-2);
+
+        return `${day}/${month}`;
```

Comment:
save some effort by using `date.toLocaleDateString(undefined, {month:"2-digit",day:"2-digit"})`

Reply:
Cool! This is better :D 
Also, I tried adding the locale. Do you think it's fine @mortenbo ?
![image](https://user-images.githubusercontent.com/32891190/176426357-bdda9717-aaf8-4dc9-8ef2-4296238b407c.png)